src/style.css

```
@@ -0,0 +1,110 @@
+body {
+  margin: 0;
+  font-family: "Roboto", "Georgia", serif;
```

```suggestion
  font-family: Roboto, sans-serif;
```
в задании не предусмотрен шрифт `Georgia`